### ISSUE TITLE
Support multiple registry

### ISSUE THREAD

**Issue by jkroepke** [Azure/docker-login#10]
Hi!

right now the actions always override the previous login.

It would be great if the auth credentials could be merged with the current, e.g.:

Here is the use case:

```
      - name: Docker Login docker.io
        uses: azure/docker-login@v1
        if: github.ref == 'refs/heads/master'
        with:
          username: ${{ secrets.REGISTRY_USERNAME }}
          password: ${{ secrets.REGISTRY_PASSWORD }}

      - name: Docker Login docker.pkg.github.com
        uses: azure/docker-login@v1
        if: github.ref == 'refs/heads/master'
        with:
          login-server: docker.pkg.github.com
          username: $GITHUB_ACTOR
          password: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by Tioborto** [comment]
+1 for us

**Comment by thesattiraju** [comment]
Closed in ref to #13
